README.md

```diff
@@ -7,16 +7,16 @@ A direct link to the IOTA C bindings directory is [here](https://github.com/iota
 
 **Environment Variable Setup**
 
-In order to execute the unit tests for the SDK, you will need to export an environment variable that points to the directory containing the C bindings once they have been compiled. 
+In order to execute the unit tests for the SDK, you will need to export an environment variable that points to the directory containing the C bindings once they have been compiled.
 As an example from my local machine, I have put the following directive into my `~/.profile`
 
 `export LD_LIBRARY_PATH=/home/developer/Documents/github/project-alvarium/alvarium-sdk-go/internal/iota/include`
 
 This variable is necessary for `make test` to execute a build prior to running the suite of unit tests
 
 # SDK Interface
-The SDK provides a minimal API -- NewSdk(), Create(), Mutate(), Transit() and BootstrapHandler().
 
+The SDK provides a minimal API -- NewSdk(), Create(), Mutate(), Transit() and BootstrapHandler().
```

Comment:
Looks like we need to add `Publish()` to this line

Reply:
fixed in 5e8f797